core/templates/dev/head/header_css_libs.html

```diff
@@ -25,5 +6,13 @@
       href="https://fonts.googleapis.com/css?family=Roboto">
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
+{% if DEV_MODE %}
+  <link rel="stylesheet" type="text/css" media="screen"
+	      href="/third_party/generated/dev/css/third_party.css">
```

Comment:
Following the existing code could you deindent this line by 5 to match the 'rel'? Same below.